src/mixins/richEditor/index.js

```diff
@@ -11,8 +11,9 @@ import stripTags from 'striptags'
 import Vue from 'vue'
 
 // Referenced from public function getMentions(): https://github.com/nextcloud/server/blob/master/lib/private/Comments/Comment.php
-// Beginning or whitespace. Non-capturing group within word boundary
-const MENTION_START = /\B(?<![^a-z0-9_\-@.'\s])/.source
+// Modified to work on MobileSafari <16.4 (see issue #5589: https://github.com/nextcloud-libraries/nextcloud-vue/issues/5589)
+// Beginning or whitespace.
```

Comment:
```suggestion
// Beginning or whitespace. Uses positive lookahead (to work on MobileSafari <16.4)
```
Since issue will be closed, we don't need to mention it (still would be trackable by Git blame / commit history)

Reply:
Well I think it makes sense to keep this to answer the question when reading the code: Why not negative lookbehind?

Reply:
I meant to keep the suggested line, not the link to resolved issue =) Thanks for rebasing